plugins/hosts/linux/cap/nfs.rb

```diff
@@ -21,32 +17,22 @@ def self.nfs_start_command(env)
 
         def self.nfs_export(env, ui, id, ips, folders)
           # Get some values we need before we do anything
-          nfs_apply_command = env.host.capability(:nfs_apply_command)
           nfs_check_command = env.host.capability(:nfs_check_command)
           nfs_start_command = env.host.capability(:nfs_start_command)
 
           nfs_opts_setup(folders)
-          output = Vagrant::Util::TemplateRenderer.render('nfs/exports_linux',
-                                           :uuid => id,
-                                           :ips => ips,
-                                           :folders => folders,
-                                           :user => Process.uid)
 
           ui.info I18n.t("vagrant.hosts.linux.nfs_export")
           sleep 0.5
 
-          nfs_cleanup(id)
```

Comment:
It was not causing a bug, but since exports are now purely atomic, there was no need to perform exports cleanup.